src/minicraft/screen/TitleDisplay.java

```diff
@@ -266,13 +246,10 @@ public void render(Screen screen) {
 		"Why is this blue?",
 		"Green is a nice color!",
 		"Red is my favorite color!",
-		"Y U NO BOAT!?",

```

Comment:
What's the reason for removing some of these?

Reply:
They're bad